dask/array/core.py

```diff
@@ -3089,6 +3089,27 @@ def broadcast_to(x, shape, chunks=None):
                  dtype=x.dtype)
 
 
+@wraps(np.broadcast_arrays)
+def broadcast_arrays(*args, **kwargs):
+    subok = bool(kwargs.pop("subok", False))
+
+    to_array = asarray
+    if subok:
+        to_array = asanyarray
```

Comment:
nit: I would prefer to write this with single assignment, `to_array = asanyarray if subok else asarray`

Reply:
Done.

Though sometimes coverage tools misreport line coverage on ternary expressions as they miss the branch.